src/TabComplete.js

```diff
@@ -307,6 +326,49 @@ class TabComplete {
             this.opts.onStateChange(this.completing);
         }
     }
+
+    _sortEntries() {
+        // largest comes first
+        const KIND_ORDER = {
+            command: 1,
+            member: 2,
+        };
+
+        this.list.sort((a, b) => {
```

Comment:
Just did a bit of digging, and indeed it is not specified in ECMAScript. Each browser implements different sorting algorithms for `Array#sort()`:
 - MergeSort for Mozilla
 - a mix of QuickSort/MergeSort/**SelectionSort**(!!!) for Chrome, depending on the types of elements inside the array.

Fun times.